packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -130,13 +132,18 @@ export class InsuredBridgeL1Client {
       ]);
 
       for (const depositRelayedEvent of depositRelayedEvents) {
-        const relayData = {
+        const relayDataForDeposit = await this.bridgePools[depositRelayedEvent.returnValues.l1Token].methods
+          .relays(depositRelayedEvent.returnValues.depositHash)
+          .call();
+        const relayData: Relay = {
+          quoteTimestamp: Number(depositRelayedEvent.returnValues.quoteTimestamp),
+          relayTimestamp: Number(relayDataForDeposit.priceRequestTime.toString()),
           depositId: Number(depositRelayedEvent.returnValues.depositId),
           sender: depositRelayedEvent.returnValues.sender,
           slowRelayer: depositRelayedEvent.returnValues.slowRelayer,
           disputedSlowRelayers: [],
           instantRelayer: ZERO_ADDRESS,
-          depositTimestamp: depositRelayedEvent.returnValues.depositTimestamp,
+          depositTimestamp: Number(depositRelayedEvent.returnValues.depositTimestamp),
```

Comment:
Not sure why TS didn't catch this before but the type of `depositTimestamp` is declared `number` in the `Relay` interface.